### PR TITLE
RHIROS-1304 handle group filtering for pdf report

### PR DESCRIPTION
**Pull request by PreetiW**
## RHIROS-1304 handle group filtering for pdf report :boom:

https://issues.redhat.com/browse/RHIROS-1304

## Why do we need this change? :thought_balloon:

Please include the __context of this change__ here.

## Documentation requires update? :memo:

- [ ] Yes
- [ ] No

## Security Checklist :lock:

Upon raising this PR please go through [RedHatInsights/secure-coding-checklist](https://github.com/RedHatInsights/secure-coding-checklist)

## :guardsman: Checklist :dart:

- [ ] Bugfix
- [x] New Feature
- [ ] Refactor
- [ ] Unittests Added
- [ ] DRY code
- [ ] Dependency Added

## Additional :mega:

Feel free to add any other relevant details such as __links, notes, screenshots__, here.

### CODE REVIEW

File: src/Components/Reports/Common/SystemsTablePage.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import styles from './styles';
 
 const columnBuilder = ({ value, style, key }) => <Text key={key} style={style}>{value}</Text>;
 
-const headerContent = ['Name', 'OS', 'CPU utilization', 'Memory utilization', 'I/O utilization', 'Suggestions', 'State', 'Last reported'];
+const headerContent = ['Name', 'Group', 'OS', 'CPU utilization', 'Memory utilization', 'I/O utilization', 'Suggestions', 'State', 'Last reported'];
```
Comment (r14chandra) [inline src/Components/Reports/Common/SystemsTablePage.js:9]: Could you include group tests -  [SystemsFirstPage.test.js](https://github.com/RedHatInsights/ros-frontend/blob/main/src/Components/Reports/Common/SystemsFirstPage.test.js)

**Review by r14chandra — CHANGES_REQUESTED** [review]

**Comment by r14chandra** [comment]
I think we would need some space between the `Name` and `Group` columns.

Screenshot from 2023-09-26 16-04-30

File: src/Components/Reports/Common/SystemsTablePage.js (hunk 1 above)
Comment (PreetiW) [reply src/Components/Reports/Common/SystemsTablePage.js:9]: @r14chandra have pushed the changes.

**Comment by PreetiW** [comment]
> I think we would need some space between the `Name` and `Group` columns.
> 
> Screenshot from 2023-09-26 16-04-30

I have fixed it and have pushed the changes.

**Review by r14chandra — APPROVED** [review]
Looks good 👍🏻 
I want to highlight one thing. I see some inconsistent width between the columns.

Screenshot from 2023-09-27 10-45-51

**Pull request merged**